src/bindings/Vscode.ml

```diff
@@ -133,6 +133,14 @@ module Workspace = struct
   external getConfiguration : string -> WorkspaceConfiguration.t
     = "getConfiguration"
     [@@bs.module "vscode"] [@@bs.scope "workspace"]
+
+  external findFiles :
+       inc:string
+    -> excl:string option
+    -> int option
```

Comment:
Is it possible to add a label to the int argument? It's not so easy to understand the purpose of it just by reading the signature.